packages/web-app-unzip/l10n/translations.json

```
@@ -1 +1 @@
-{}
+{"de":  {"Unzip":"Unzip","Extract here":"Entpacken","Password protected archives cannot be extracted":"Passwortgeschützte Archive können nicht entpackt werden","Failed to extract archive":"Entpacken fehlgeschlagen","Archive exceeds the maximum size of %{maxSize}":"Das Archiv überschreitet die maximale Größe von %{maxSize}"}}
```

Not sure what will happen to blanks, but better be on the safe side I'd say 🤔 

```suggestion
{"de":{"Unzip":"Unzip","Extract here":"Entpacken","Password protected archives cannot be extracted":"Passwortgeschützte Archive können nicht entpackt werden","Failed to extract archive":"Entpacken fehlgeschlagen","Archive exceeds the maximum size of %{maxSize}":"Das Archiv überschreitet die maximale Größe von %{maxSize}"}}
```